hw/opentitan/ot_spi_host.c

```diff
@@ -1235,9 +1232,9 @@ static const MemoryRegionOps ot_spi_host_ops = {
 
 static Property ot_spi_host_properties[] = {
     DEFINE_PROP_STRING("ot_id", OtSPIHostState, ot_id),
-    DEFINE_PROP_UINT8("num-cs", OtSPIHostState, num_cs, 1),
-    DEFINE_PROP_UINT32("bus-num", OtSPIHostState, bus_num, 0),
-    DEFINE_PROP_UINT32("pclk", OtSPIHostState, pclk, 0),
+    DEFINE_PROP_UINT32("num-cs", OtSPIHostState, num_cs, 1u),
+    DEFINE_PROP_UINT32("bus-num", OtSPIHostState, bus_num, 0u),
+    DEFINE_PROP_UINT32("pclk", OtSPIHostState, pclk, 0u),
```

Comment:
I think we usually don't put `u` suffix on zero constants

Reply:
Let's keep it like this. As discussed offline, our convention originated in a compiler bug that is not relevant anymore.